app/helpers/validation_helper.php

```diff
@@ -28,4 +28,10 @@ function is_password_valid($check)
 function is_email_valid($check)
 {
 	return preg_match("/^([a-z0-9_\.-]+\@[\da-z\.-]+\.[a-z\.]{2,6})$/", $check);
+}
+
+function is_valid($check)
+{
+	return preg_match("/^[a-zA-Z]+(?:[\s][A-Za-z]+)*$/", $check); //Name should only contain letters and one or more
+												//non consecutive space
```

Comment:
This function po will match first name and last name containing one or more space but not consecutive po